content/en/synthetics/settings/_index.md

```diff
@@ -62,7 +62,19 @@ You can create variables from your existing [HTTP tests][1] by parsing their ass
 
 {{< img src="synthetics/settings/variable_fromhttp_3.png" alt="Variable from HTTP Test" style="width:80%;">}}
 
-**Note:** Variable values are updated whenever the test they are extracted from runs.
+In addition to extracting a value with a `regex`, you can also:
```

Comment:
Can you rephrase this to be a full sentence? Full sentences are easier for the translators to localize. I attempted to rephrase this, but I'm not confident of the context and meaning
```suggestion
In addition to extracting a value with a regex, you can also use a regex to parse the following examples:
```